backend-models/courseedService_microservices.txt

```diff
@@ -0,0 +1,79 @@
+courseedService_microservices
+
+
+Functionality : 
+
+Row 40 :If isset($_SESSION)
+Checks what kind of user it is, meaning, student or teacher, if none, get guest. 
+You get username based on your, "uid".
+Name:userCheck
+
+Row 49 : Gets the username based on "uid" for logging.
+Name : getUsername
+
+Row 54 : If "uid" exists, set username. 
+Name : setUser
+
+Row 67 : checklogin(). Checks what kind of user it is based on "uid". Micro
```

Comment:
doesn't it just check if its true?
this is the function being accessed.

function checklogin()
{
      // If neither session nor post return not logged in
      if(array_key_exists('loginname', $_SESSION)){
        return true;
      } else if(failedLoginCount($_SERVER['REMOTE_ADDR']) < 100 && array_key_exists('username', $_COOKIE) && array_key_exists('password', $_COOKIE)) {
        return login($_COOKIE['username'], $_COOKIE['password'], false);
      } else {
        return false;
      }
}

you might be mixing this up with the logging that also takes place in the file :)

Reply:
![image](https://user-images.githubusercontent.com/129070844/231104403-36b6ad84-0ee8-4dd2-9acd-d6d925166a6d.png)
This is a separate function in another file (sessions.php). It is described there, we can skip it for this issue.